src/sharedStyles/TextStyles.ts

```diff
@@ -1,6 +1,13 @@
 import { FileFormat1 as FileFormat } from '@sketch-hq/sketch-file-format-ts';
```

Comment:
This one is better reviewed side-by-side. The change is trivial, but the diff that git produces is horrible